pkg/intermediate/aggregate.go

```diff
@@ -215,6 +215,71 @@ func (a *AggregationProcess) GetExpiryFromExpirePriorityQueue() time.Duration {
 	return a.inactiveExpiryTimeout
 }
 
+// GetRecordStrings returns the flow records by FlowKey from flowKeyRecordMap.
+// Returns all the flow records if FlowKey is not provided.
+func (a *AggregationProcess) GetRecordStrings(flowKey *FlowKey) []string {
```

Comment:
I think GetRecords is sufficient. In the function description, you can add in string format.

Reply:
Updated